store/postgres/src/chain_store.rs

```diff
@@ -1700,7 +1700,7 @@ impl ChainStoreTrait for ChainStore {
             block_ptr.number >= offset,
             "block offset {} for block `{}` points to before genesis block",
             offset,
-            block_ptr.hash_hex()
+            format!("{:x}", block_ptr)
```

Comment:
I'd rather not repeat format everywhere, prolly better just adding a fn for this.

Reply:
Reverted to just changing the line pointed out above and using the `fn` as it was. Thanks